src/lib/index.module.css

```diff
@@ -0,0 +1,166 @@
+.layout {
```

Comment:
Please adjust index.module.css to include styles your component needs, removing those used by react-supervenn